src/test/java/rest/salesforce/org/steps/AccountsApiSteps.java

```diff
@@ -9,20 +9,56 @@
 
 package rest.salesforce.org.steps;
 
+import api.ApiManager;
+import api.ApiMethod;
+import api.ApiRequestBuilder;
+import api.ApiResponse;
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import entities.Account;
+import entities.responseobject.ResponseObject;
 import io.cucumber.java.After;
 import io.cucumber.java.Before;
 import io.cucumber.java.en.And;
 import io.cucumber.java.en.Given;
 import io.cucumber.java.en.Then;
 import io.cucumber.java.en.When;
+import org.testng.Assert;
+
+import static configfile.Configuration.dotenv;
 
 public class AccountsApiSteps {
+
+    private ApiRequestBuilder requestBuilder = new ApiRequestBuilder();
+    private ResponseObject responseObject = new ResponseObject();
+    private ApiResponse apiResponse = new ApiResponse();
+    Account accountToSend = new Account();
+
     @Before("@CreateAndDeleteAccount")
-    public void createAnAccount() {
+    public void createAnAccount() throws JsonProcessingException {
+        accountToSend.setName("cristian choque desde java");
```

Comment:
change the name given to a more general name, and use english

Reply:
ok